library_api/main.go

```diff
@@ -5,11 +5,23 @@ import (
 	"library_api/router"
 	"log"
 	"os"
+	"fmt"
+
+	"gorm.io/gorm"
+  	"gorm.io/driver/postgres"
 )
 
 
 func main() {
-	_, err := config.NewJsonConfig("config.json")
+	dsn := "host=postgres user=myuser password=mypassword dbname=mydatabase port=5432 sslmode=disable"
+	_, err := gorm.Open(postgres.Open(dsn), &gorm.Config{})
```

Comment:
you provide config by env variables but don't retrieve them inside code, please do it

Reply:
os.GetEnv